docs/doxygen-user/live_triage.dox

```diff
@@ -0,0 +1,33 @@
+/*! \page live_triage_page Live Triage
+
+\section live_triage_overview Overview
+
+The Live Triage feature allows you to load Autopsy onto a removable drive to run on target systems while making minimal changes to that target system. This will currently only work on Windows systems.
+
+\section live_triage_create_drive Creating a live triage drive
+
+To create a live triage drive, go to Tools->Make Live Triage Drive to bring up the main dialog.
+
+\image html live_triage_dialog.png
+
+Select the drive you want to use - any type of USB storage device will work. For best results use the fastest drive available. Once the process is complete the root folder will contain an Autopsy folder and a RunFromUSB.bat file. 
+
+\section live_triage_usage Running Autopsy from the live triage drive
+
+Insert the drive into the target machine and browse to it in Windows Explorer. Right click on RunFromUSB.bat and select "Run as administrator". This is necessary to analyze the local drives. 
+
+\image html live_triage_script.png
+
+Running the script will generate a few more directories on the USB drive. configData stores all the data used by Autopsy - primarily configuration files and temporary files. You can make changes to the Autopsy settings and they will persist between runs. The cases directory is created as a recommended place to save your case data. You will need to browse to it when creating a case in Autopsy. 
```

Comment:
It seems a little awkward to start a sentenc with the loser case "configData." Maybe "The configData directory..."?